straxen/plugins/acqmon_processing.py

```diff
@@ -109,15 +188,26 @@ def compute(self, aqmon_hits, start, end):
                     if inx == len(veto_hits_stop['time']):
                         continue
                     else:
-                        result['veto_interval'][vetos_seen] = veto_hits_stop['time'][inx] - time
                         result["time"][vetos_seen] = time
                         assert time < veto_hits_stop['time'][inx]
                         result["endtime"][vetos_seen] = veto_hits_stop['time'][inx]
-                        result["veto_type"][vetos_seen] = name + 'veto'
+                        result["veto_type"][vetos_seen] = veto_name + 'veto'
                         vetos_seen += 1
+
+        # Straxen deadtime is special, it's a start and stop with no data
+        # but already an interval so easily used here
+        artificial_deadtime = aqmon_hits[(aqmon_hits['channel'] ==
+                                          AqmonChannels.ARTIFICIAL_DEADTIME)]
+        n_artificial = len(artificial_deadtime)
+
+        if n_artificial:
+            result[vetos_seen:n_artificial]['time'] = artificial_deadtime['time']
+            result[vetos_seen:n_artificial]['endtime'] = strax.endtime(artificial_deadtime)
+            result[vetos_seen:n_artificial]['veto_type'] = 'straxen_deadtime_veto'
+            vetos_seen += n_artificial
+
         result = result[:vetos_seen]
-        result['time'] = np.clip(result['time'], start, end)
-        result['endtime'] = np.clip(strax.endtime(result), 0, end)
+        result['veto_interval'] = result['endtime'] - result['time']
```

Comment:
I think it does not really matter, but this is somewhat redundant is not it?

Reply:
fully redundant, but since it's such a light weight dtype, I would just leave it because it's the field that is mostly used for this dtype